tests/testing.py

```diff
@@ -4,14 +4,22 @@
 import spur.ssh
 
 
+def _int_or_none(val):
+    return int(val) if val is not None else None
+
+
+HOSTNAME=os.environ.get("TEST_SSH_HOSTNAME", "127.0.0.1")
+USERNAME=os.environ["TEST_SSH_USERNAME"]
+PASSWORD=os.environ["TEST_SSH_PASSWORD"]
+PORT=_int_or_none(os.environ.get("TEST_SSH_PORT"))
+
+
 def create_ssh_shell(missing_host_key=None, shell_type=None):
-    port_var = os.environ.get("TEST_SSH_PORT")
-    port = int(port_var) if port_var is not None else None
     return spur.SshShell(
-        hostname=os.environ.get("TEST_SSH_HOSTNAME", "127.0.0.1"),
-        username=os.environ["TEST_SSH_USERNAME"],
-        password=os.environ["TEST_SSH_PASSWORD"],
-        port=port,
+        hostname=HOSTNAME,
+        username=USERNAME,
+        password=PASSWORD,
+        port=PORT,
         missing_host_key=(missing_host_key or spur.ssh.MissingHostKey.accept),
-        shell_type=shell_type,
```

Comment:
Any reason for getting rid of the trailing comma? I normally keep them in for multi-line arguments for consistency.